temporalcli/commands.operator_namespace.go

```diff
@@ -0,0 +1,335 @@
+			UpdateInfo:        updateInfo,
+			Config:            updateConfig,
+			ReplicationConfig: replicationConfig,
+		}
+	}
+
+	if c.Verbose {
+		_ = cctx.Printer.PrintStructured(updateRequest, printer.StructuredOptions{})
+	}
+
+	_, err = cl.WorkflowService().UpdateNamespace(cctx, updateRequest)
+	if err != nil {
+		switch err.(type) {
+		case *serviceerror.NamespaceNotFound:
+			return err
+		default:
+			return fmt.Errorf("namespace update failed: %w", err)
+		}
+	} else {
+		cctx.Printer.Println(color.GreenString("namespace %s update succeeded.", nsName))
+	}
+	return nil
+}
+
+func printNamespaces(cctx *CommandContext, responses ...*workflowservice.DescribeNamespaceResponse) error {
+	namespaces := make([]map[string]any, len(responses))
+	for i, resp := range responses {
+		namespaces[i] = map[string]any{
+			"NamespaceInfo.Name":                   resp.NamespaceInfo.Name,
+			"NamespaceInfo.Id":                     resp.NamespaceInfo.Id,
+			"NamespaceInfo.Description":            resp.NamespaceInfo.Description,
+			"NamespaceInfo.OwnerEmail":             resp.NamespaceInfo.OwnerEmail,
+			"NamespaceInfo.State":                  resp.NamespaceInfo.State,
+			"NamespaceInfo.Data":                   resp.NamespaceInfo.Data,
+			"Config.WorkflowExecutionRetentionTtl": resp.Config.WorkflowExecutionRetentionTtl.AsDuration(),
+			"ReplicationConfig.ActiveClusterName":  resp.ReplicationConfig.ActiveClusterName,
+			"ReplicationConfig.Clusters":           resp.ReplicationConfig.Clusters,
+			"Config.HistoryArchivalState":          resp.Config.HistoryArchivalState,
+			"Config.VisibilityArchivalState":       resp.Config.VisibilityArchivalState,
+			"IsGlobalNamespace":                    resp.IsGlobalNamespace,
+			"FailoverVersion":                      resp.FailoverVersion,
+			"FailoverHistory":                      resp.FailoverHistory,
+			"Config.HistoryArchivalUri":            resp.Config.HistoryArchivalUri,
+			"Config.VisibilityArchivalUri":         resp.Config.VisibilityArchivalUri,
+		}
+	}
+
+	return cctx.Printer.PrintStructured(
+		namespaces,
+		printer.StructuredOptions{
+			Fields: []string{
+				"NamespaceInfo.Name", "NamespaceInfo.Id", "NamespaceInfo.Description",
+				"NamespaceInfo.OwnerEmail", "NamespaceInfo.State", "NamespaceInfo.Data",
+				"Config.WorkflowExecutionRetentionTtl", "ReplicationConfig.ActiveClusterName",
+				"ReplicationConfig.Clusters", "Config.HistoryArchivalState", "Config.VisibilityArchivalState",
+				"IsGlobalNamespace", "FailoverVersion", "FailoverHistory", "Config.HistoryArchivalUri",
+				"Config.VisibilityArchivalUri",
+			},
+		},
+	)
+}
+
+func archivalState(input string) enums.ArchivalState {
+	switch input {
+	case "disabled":
+		return enums.ARCHIVAL_STATE_DISABLED
+	case "enabled":
+		return enums.ARCHIVAL_STATE_ENABLED
+	}
+	return enums.ARCHIVAL_STATE_UNSPECIFIED
+}
+
+func getNamespaceFromIDArgs(cctx *CommandContext, nsID string, args []string) (string, string, error) {
```

Comment:
Can you inline this instead of making a single-use package-level method for all?